transport/src/main/java/io/netty/channel/DefaultChannelId.java

```diff
@@ -53,7 +53,10 @@
 
     private static final AtomicInteger nextSequence = new AtomicInteger();
 
-    static ChannelId newInstance() {
+    /**
+     * Create and return a new {@link ChannelId} instance.
+     */
+    public static ChannelId newInstance() {
```

Comment:
Should return `DefaultChannelId` given that this method is the factory method of `DefaultChannelId`?